assets/multiqc_config.yaml

```diff
@@ -24,3 +25,5 @@ report_section_order:
 table_columns_visible:
     FastQC:
         percent_duplicates: False
+
+export_plots: true
```

Comment:
Didn't https://github.com/nf-core/rnaseq/issues/200 say that this was changed to `True` with a capital `T`?

Note that I'm kind of surprised that the capitalisation should make any difference, based on the [YAML specs](https://yaml.org/type/bool.html)

Reply:
I double checked the output from test and found the PDF, SVG and PNG files in place - so I think this is just fine 👍